src/servicebus/azext_servicebus/tests/test_servicebus_commands.py

```diff
@@ -0,0 +1,493 @@
+
+        # Create Namespace - Secondary
+        self.cmd('servicebus namespace create --resource-group {rg} --name {namespacenamesecondary}'
+                 ' --location {loc_north} --tags {tags} --sku-name {sku} --sku-tier {tier}',
+                 checks=[self.check('sku.name', self.kwargs['sku'])])
+
+        # Get Created Namespace - Secondary
+        getnamespace2result = self.cmd('servicebus namespace show --resource-group {rg} --name {namespacenamesecondary}',
+                                       checks=[self.check('sku.name', self.kwargs['sku'])]).output
+
+        # Create Authoriazation Rule
+        self.cmd('servicebus namespace authorizationrule create --resource-group {rg} --namespace-name {namespacenameprimary} --name {authoname} --access-rights {accessrights}', checks=[self.check('name', self.kwargs['authoname'])])
+
+        partnernamespaceid = json.loads(getnamespace2result)['id']
+        self.kwargs.update({'id': partnernamespaceid})
+        # Get Create Authorization Rule
+        self.cmd('servicebus namespace authorizationrule show --resource-group {rg} --namespace-name {namespacenameprimary}'
+                 ' --name {authoname}', checks=[self.check('name', self.kwargs['authoname'])])
+
+        # CheckNameAvailability - Alias
+
+        self.cmd('servicebus georecovery-alias check-name-availability --resource-group {rg} --namespace-name {namespacenameprimary} --name {aliasname}', checks=[self.check('nameAvailable', True)])
+
+        # Create alias
+        self.cmd('servicebus georecovery-alias create  --resource-group {rg} --namespace-name {namespacenameprimary} --alias {aliasname} --partner-namespace {id}')
+
+        # get alias - Primary
+        self.cmd('servicebus georecovery-alias show  --resource-group {rg} --namespace-name {namespacenameprimary} --alias {aliasname}')
+
+        # get alias - Secondary
+        self.cmd('servicebus georecovery-alias show  --resource-group {rg} --namespace-name {namespacenamesecondary} --alias {aliasname}')
+
+        getaliasprimarynamespace = self.cmd(
+            'servicebus georecovery-alias show  --resource-group {rg} --namespace-name {namespacenameprimary} --alias {aliasname}').output
+
+        # check for the Alias Provisioning succeeded
+        while json.loads(getaliasprimarynamespace)['provisioningState'] != ProvisioningStateDR.succeeded.value:
+            time.sleep(30)
+            getaliasprimarynamespace = self.cmd(
+                'servicebus georecovery-alias show  --resource-group {rg} --namespace-name {namespacenameprimary} --alias {aliasname}').output
+
+        # Break Pairing
+        self.cmd('servicebus georecovery-alias break-pairing  --resource-group {rg} --namespace-name {namespacenameprimary}'
+                 ' --alias {aliasname}')
+
+        getaliasafterbreak = self.cmd(
+            'servicebus georecovery-alias show  --resource-group {rg} --namespace-name {namespacenameprimary} --alias {aliasname}').output
```

Comment:
You can use `get_output_in_json()` so no need to do the json.loads.

Reply:
Resolved